src/skyss0fly/WelcomeMessage/Main.php

```diff
@@ -49,9 +58,10 @@ public function onPlayerLeave(PlayerQuitEvent $event): void {
         if ($leaveMessageEnabled === true) {
```

Comment:
This is redundant, you can simplify the condition.

```suggestion
        if ($leaveMessageEnabled) {
```